man/elixhauser_comorbidity_index.Rd

```diff
@@ -52,7 +53,10 @@ This affects comorbidities present in a patient with different degrees of severi
 }
 \references{
 \itemize{
-\item{Elixhauser A, et al. Med Care, 1998. https://doi.org/10.1097/MLR.0b013e31819432e5}
+\item{Elixhauser A, et al. Med Care, 1998. https://www.jstor.org/stable/3766985}
+\item{Quen H, et al. Med Care, 2005. https://www.jstor.org/stable/3768193}
+\item{Van Walraven C, et al. Med Care, 2009. https://doi.org/10.1097/MLR.0b013e31819432e5}
```

Comment:
Hi Yishan, I saw Vaakesan commented this paper in the description. But I don't see other paper reference in the code review description. From my understanding, the request is to update the reference from 1998 paper to 2009 paper from Van Walraven for Elixhauser  method? Please correct me if I'm wrong. I didn't find any instructions about charlson score paper references. Are those being communicated offline?

Reply:
Hi Shirly, Thank you for your review! 
The methods in the functions (both Charlson and Elixhauser) are built on top of the three papers here. The raw comorbidities were developed in 1998 paper, the mapping from ICD10 to the Elixhauser comorbidities were developed in 2005 paper and the weights to compute a single integer score was derived in 2009 paper. I think it would be better to list them all. You can find more details in [this round table presentation slides deck](https://docs.google.com/presentation/d/1bux3tDlk3vSYsBbF-TkeYOZoOey4aS9Q/edit#slide=id.p1)